Assets/LeapMotion/Scripts/Utils/Hands.cs

```diff
@@ -0,0 +1,169 @@
+﻿using UnityEngine;
+using System.Collections.Generic;
+using UnityEngine.SceneManagement;
+
+namespace Leap.Unity {
+
+  public static class Hands {
+
+    private static LeapServiceProvider s_provider;
+    private static GameObject s_leapRig;
+
+    static Hands() {
+      InitStatic();
+      SceneManager.activeSceneChanged += InitStaticOnNewScene;
+    }
+
+    private static void InitStaticOnNewScene(Scene unused, Scene unused2) {
+      InitStatic();
+    }
+
+    private static void InitStatic() {
+      s_provider = GameObject.FindObjectOfType<LeapServiceProvider>();
+      s_leapRig = s_provider.transform.parent.parent.parent.gameObject;
+    }
+
+    /// <summary>
+    /// Static convenience accessor for the Leap Rig GameObject.
+    /// </summary>
+    public static GameObject Rig {
```

Comment:
imo it should step up the hierarchy until it finds an object with a Camera, and then return that one's parent.

And then cache it because that many GetComponents() every frame will annihilate your performance!